app/helpers/content_substitutor_helper.rb

```diff
@@ -0,0 +1,15 @@
+module ContentSubstitutorHelper
+  def content_substitutor
+    @content_substitutor ||= ContentSubstitutor.new(
+      service_name: service.service_name,
+      reference_number_enabled: reference_number_enabled
+    )
+  end
+
+  def reference_number_enabled
```

Comment:
```suggestion
  def reference_number_enabled?
```

one to add in the next PR maybe